lib/src/view/design_side_helix.dart

```diff
@@ -61,6 +62,7 @@ class DesignSideHelixComponent extends UiComponent2<DesignSideHelixProps> with P
     // of idx, which is useful for making figures in the documentation showing how the grids work
 //    bool SHOW_HELIX_COORDINATES_INSTEAD_OF_IDX = true;
     bool SHOW_HELIX_COORDINATES_INSTEAD_OF_IDX = false;
+    bool SHOW__INDEX_AND_COORDINATES = props.show_grid_coordinates;
```

Comment:
Don't make this intermediate variable `SHOW__INDEX_AND_COORDINATES`.

The convention is that all-caps things like this are constants that don't change while the program is running. Some of them are constants used in normal running of the program (in lib/src/constants.dart) and others are used for us to temporarily test something (such as `SHOW_HELIX_COORDINATES_INSTEAD_OF_IDX`, which I used to make some figures, or `USE_REDUX_DEV_TOOLS` and `RUN_TEST_CODE_INSTEAD_OF_APP` at the top of app.dart). Just use `props.show_grid_coordinates` directly wherever it is needed.

You'll find some code that assigns props variables into local variables, e.g., `var helix = props.helix;`, but in general I prefer to reference `props` variables directly (i.e., just use `props.helix`), which helps to see where the data is coming from. This is old code that I used to adapt from an earlier version (prior to using React) where there were already local variables, and it was easier simply to initialize them differently, than to get rid of them and replace them.